### ISSUE TITLE
No startup: Missing database port

### ISSUE THREAD

**Issue by Floyer007** [bitnami/bitnami-docker-phpbb#74]
**Description**

The Docker-Container doesn't start anymore due to missing database port after upgrade to :latest (3.3.5), but it turned out even other versions (3.3.1 doesn't work anymore).
I used the exact same docker-compose file and got a working older 3.3.1-debian-10-r22 version I used before and a non working latest 3.3.1.
My goal is to update to the current newest 3.3.5

**Steps to reproduce the issue:**

1. Pull the newest 3.3.1 image
2. See the log "docker logs forum_phpbb"

**Describe the results you received:**

Log output is as followed:
```
phpbb 11:17:27.34 
phpbb 11:17:27.34 Welcome to the Bitnami phpbb container
phpbb 11:17:27.34 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-phpbb
phpbb 11:17:27.34 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-phpbb/issues
phpbb 11:17:27.34 
phpbb 11:17:27.35 INFO  ==> ** Starting phpBB setup **
phpbb 11:17:27.37 INFO  ==> Configuring PHP options
phpbb 11:17:27.37 INFO  ==> Setting PHP expose_php option
phpbb 11:17:27.38 INFO  ==> Validating settings in MYSQL_CLIENT_* env vars
phpbb 11:17:27.45 INFO  ==> Restoring persisted phpBB installation
/opt/bitnami/scripts/libphpbb.sh: line 289: 2: missing database port
```
This error doesn't appear with "bitnami/phpbb:3.3.1-debian-10-r22". Other newer might work, but are not tested.
With the current "bitnami/phpbb:3.3.1" the board doesn't work anymore and the previous log shows up.

**Describe the results you expected:**

Working board without error within log.

**Version**

- Output of `docker version`:

```
Client: Docker Engine - Community
 Version:           19.03.9
 API version:       1.40
 Go version:        go1.13.10
 Git commit:        9d988398e7
 Built:             Fri May 15 00:25:25 2020
 OS/Arch:           linux/amd64
 Experimental:      false1

Server: Docker Engine - Community
 Engine:
  Version:          19.03.9
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.10
  Git commit:       9d988398e7
  Built:            Fri May 15 00:23:57 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.21.0, build unknown
docker-py version: 3.4.1
CPython version: 3.7.3
OpenSSL version: OpenSSL 1.1.1d  10 Sep 2019
```
- Docker-compose file:
```yaml
version: '2'
services:
  phpbb:
    image: 'bitnami/phpbb:3.3.1'
    container_name: forum_phpbb
    restart: always
    environment:
      - MARIADB_HOST=db
      - MARIADB_PORT_NUMBER=3306
      - PHPBB_HOST=https://MYURL/forum/
      - PHPBB_DATABASE_USER=#myuser#
      - PHPBB_DATABASE_PASSWORD=#mypassword#
      - PHPBB_DATABASE_NAME=forum
    ports:
      - '8081:80'
    volumes:
      - './phpbb:/bitnami/phpbb'
    depends_on:
      - db
  db:
    image: mariadb:latest
    container_name: forum_db
    restart: always
    environment:
      - MYSQL_DATABASE=forum
      - MYSQL_USER=#myuser#
      - MYSQL_PASSWORD=#mypassword#
      - MYSQL_RANDOM_ROOT_PASSWORD=yes
    volumes:
      - ./db-data:/var/lib/mysql
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**
VM with docker.

**Comment by carrodher** [comment]
Please, [note the Bitnami team is not actively monitoring GitHub Issues for this project](https://github.com/bitnami/bitnami-docker-phpbb#community-supported-solution). This issue will stay open for 20 days to allow the community to contribute, after 15 days without activity the issue will be marked as stale being closed after 5 days.

In this specific case there are no recent changes, note the last change in the script returning the error was 7 months ago, see https://github.com/bitnami/bitnami-docker-phpbb/commits/master/3/debian-10/rootfs/opt/bitnami/scripts/libphpbb.sh

If you would like to contribute to this container image by fixing a bug or adding a new feature feel free to create a PR, the Bitnami team will be happy to review it and provide feedback.

**Comment by Floyer007** [comment]
Thank you for the update.

I figured out, that actually r49 still works.
r50 and greater cause the error "missing database port".

I debugged a bit and figured out the root cause.
The config.php wasn't updated automatically, other than said within the header:
`// phpBB 3.0.x auto-generated configuration file
  3 // Do not change anything in this file!`
After changing `$dbport = '';` to `$dbport = '3306';` everything works fine now.
